.eslintrc.js

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-const { escapeRegExp, map } = require( 'lodash' );
+const { escapeRegExp } = require( 'lodash' );
```

Comment:
That's surprising, it should error in master...

Reply:
```
$ ./node_modules/.bin/eslint .eslintrc.js

/Users/pascalb/Workspace/WordPress/gutenberg/.eslintrc.js
  0:0  warning  File ignored by default.  Use a negated ignore pattern (like "--ignore-pattern '!<relative/path/to/filename>'") to override

✖ 1 problem (0 errors, 1 warning)
```

That explains it :-)

Reply:
I was wondering the same, but reason is that the `.eslintrc.js` file is, by default, ignored. So ESLint doesn't check it.

Reply:
Nice, good to know 😃

Reply:
> I was wondering the same, but reason is that the `.eslintrc.js` file is, by default, ignored. So ESLint doesn't check it.

Can we change this? Not immediately obvious why we'd want it to be exempt.

Reply:
Sure, I'll create a PR for that.